pkgs/development/python-modules/apache-airflow/default.nix

```diff
@@ -58,24 +65,32 @@
 , tabulate
 , tenacity
 , termcolor
+, typing-extensions
 , unicodecsv
 , werkzeug
 , pytestCheckHook
 , freezegun
 , mkYarnPackage
 }:
 let
-  version = "2.3.3";
+  version = "2.3.4";
 
   airflow-src = fetchFromGitHub rec {
     owner = "apache";
     repo = "airflow";
     rev = "refs/tags/${version}";
-    sha256 = "sha256-N+6ljfSo6+UvSAnvDav6G0S49JZ1VJwxmaiKPV3/DjA=";
+    # Required because the GitHub archive tarballs don't appear to include tests
+    leaveDotGit = true;
+    sha256 = "sha256-rxvLyz/hvZ6U8QKy9MiVofU0qeeo7OHctAj2PkxLh2c=";
   };
 
   # airflow bundles a web interface, which is built using webpack by an undocumented shell script in airflow's source tree.
   # This replicates this shell script, fixing bugs in yarn.lock and package.json
+  # To update yarn.lock and package.json:
+  # cd pkgs/development/python-modules/apache-airflow
+  # curl -O https://raw.githubusercontent.com/apache/airflow/$version/airflow/www/yarn.lock
+  # curl -O https://raw.githubusercontent.com/apache/airflow/$version/airflow/www/package.json
+  # yarn2nix > yarn.nix
```

Comment:
Could this become a `passthru.updateScript`?

Reply:
I'll have a look, thanks.